packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -4487,6 +4780,203 @@ describe('TransactionController', () => {
     });
   });
 
+  describe('initTrackingMap', () => {
+    it('doesnt get called if the feature flag is disabled', () => {
+      const hub = new EventEmitter() as TransactionControllerEventEmitter;
+      const spy = jest.fn();
+      hub.on('tracking-map-init', spy);
```

Comment:
Agreed. We will remove these tests. The tests in the new test file `TransactionsControllerIntegration.test.ts` are actually targeting effects and function rather than implementation.

Reply:
Currently working on ensuring that all paths in these tests are functionally covered.